code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -229,15 +235,18 @@
 	new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
 
 /obj/item/storage/box/syndie_kit/emp
-	name = "boxed EMP kit"
+	name = "box"
 
 /obj/item/storage/box/syndie_kit/emp/PopulateContents()
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/implanter/emp(src)
+	for(var/i in 1 to 5)
```

Comment:
does this randomize the amount of grenades you get in the box? What purpose does that serve except to screw people over?

Reply:
Heck no, it just means in items slots 1 through 5, spawn an EMP grenade here